pkg/consts/consts.go

```diff
@@ -32,3 +32,9 @@ const (
 	NetworkOperatorResourceNamespace = "mlnx-network-operator-resources"
 	NicClusterPolicyResourceName     = "nic-cluster-policy"
 )
+
+const (
```

Comment:
Should be the same naming convention as the above

Reply:
chaged